### PR TITLE
Refactor AddReadingPresenter

### PR DESCRIPTION
**Pull request by jeonsoh**
- setReadingTimeNow and updateReadingSplitDateTime have duplicated codes
- getReadingCal and getReadingTime have same problem

### CODE REVIEW

**Comment by emartynov** [comment]
Thank you for the contribution!

Are these methods ever used? I see you changed methods itself but I don't see you changed places where they were used.

**Comment by raacker** [comment]
Seems removed duplicated codes. There are some codes that have totally same functionality.

Seems Okay :)

**Comment by jeonsoh** [comment]
Yes. raacker is right. 
I tend to just remove duplicated codes. So, not effect on any other codes.

File: app/src/main/java/org/glucosio/android/presenter/AddReadingPresenter.java [hunk 1]
```diff
@@ -40,10 +27,9 @@ public void updateReadingSplitDateTime(Date readingDate) {
         this.readingMonth = splitDateTime.getMonth();
     }
 
-    public Date getReadingTime() {
-        Calendar cal = Calendar.getInstance();
-        cal.set(Integer.parseInt(readingYear), Integer.parseInt(readingMonth) - 1, Integer.parseInt(readingDay), Integer.parseInt(readingHour), Integer.parseInt(readingMinute));
-        return cal.getTime();
+    public void setReadingTimeNow() {
+        Date formatted = Calendar.getInstance().getTime();
```
Comment (emartynov) [inline app/src/main/java/org/glucosio/android/presenter/AddReadingPresenter.java]: Just `new Date()` is enough

**Comment by jeonsoh** [comment]
Okay! I modified that part, changed to new Date(). 
Thank you very well!

**Pull request merged**